panimg/models.py

```diff
@@ -5,13 +5,24 @@
 from enum import Enum
 from functools import cached_property
 from pathlib import Path
-from typing import Any, Dict, FrozenSet, List, NamedTuple, Optional, Set, Tuple
+from typing import (
+    Any,
+    ClassVar,
+    Dict,
+    FrozenSet,
+    List,
+    NamedTuple,
+    Optional,
+    Set,
+    Tuple,
+)
 from uuid import UUID, uuid4
 
 import numpy as np
 import SimpleITK
-from pydantic import BaseModel, ConfigDict, field_validator
+from pydantic import BaseModel, ConfigDict
 from pydantic.dataclasses import dataclass
+from pydantic.functional_validators import field_validator
```

Comment:
Why the changes to this file? I don't see why the import needs to be moved or why the type annotations need to be added.

Reply:
That was it, mypy was outdated. Updated it and reverted this change.